openstack/tests/test_openstack.py

```diff
@@ -0,0 +1,297 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
```

Comment:
I think the license needs to be updated

Reply:
Good catch, updated them in all the test files.